src/web/utils/parserUtils.ts

```diff
@@ -167,7 +168,8 @@ function parseRangesToHTMLNodes(
     return {dom: rootElement, tree: rootNode};
   }
 
-  const markdownRanges = ungroupRanges(ranges);
+  const sortedRanges = sortRanges(ranges);
+  const markdownRanges = ungroupRanges(sortedRanges);
```

Comment:
Why don't we move `ungroupRanges` to `rangeUtils.ts` as well?

Reply:
Done in [0d710a0](https://github.com/Expensify/react-native-live-markdown/pull/585/commits/0d710a0dcd7577aec7fd492aec3b8dd8abfdb117)